contracts/BallotsStorage.sol

```diff
@@ -80,9 +80,9 @@ contract BallotsStorage is EternalStorage, EnumBallotTypes, EnumKeyTypes, EnumTh
     ) public onlyOwner {
         require(!initDisabled());
         require(_thresholds.length == uint256(ThresholdTypes.MetadataChange));
-        require(_thresholds.length <= 255);
+        require(_thresholds.length < 255);
         for (uint8 thresholdType = uint8(ThresholdTypes.Keys); thresholdType <= _thresholds.length; thresholdType++) {
-            uint256 thresholdValue = _thresholds[thresholdType - 1];
+            uint256 thresholdValue = _thresholds[thresholdType - uint8(ThresholdTypes.Keys)];
```

Comment:
I think this won't help much: we're still assuming that `ThresholdTypes.Keys` is `1` and we still don't have guarantees from solidity on order and numeric values of enum members.

Reply:
@Eenae Maybe it is better for us not to use enums?

Reply:
It's up to you. I'm just saying that this particular change does't help much.

Reply:
@Eenae Solidity docs tell that valid values for `enum` of `n` members are `0 until n - 1`: https://solidity.readthedocs.io/en/v0.4.24/miscellaneous.html#internals-cleaning-up-variables - so, `enum` in Solidity seems to be the same as in other programming languages. That still doesn't guarantee the order of enum values, but I think everything is ok with it because unit tests show that.